models/cam-fv/model_mod.f90

```diff
@@ -3584,7 +3584,8 @@ subroutine init_discard_high_obs()
 ! incoming observations which have vertical units of scale height.
 ! so far we have localized in scale height but never had obs
```

Comment:
this comment is misleading and should be updated at some point, since you're moving the call to 'obs_too_high()' to the get_close routines.  the call in model_interpolate() hasn't seen incoming obs with scale height as vertical, but that's getting removed.  at get_close() time the obs are converted to the vertical localization type, so definitely yes there will be obs that have this type.